src/interface/src/app/treatments/treatmentplan-about-tab/treatmentplan-about-tab.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { AsyncPipe, NgIf, DecimalPipe } from '@angular/common';
+import { TreatmentsState } from '../treatments.state';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { Scenario } from '@types';
+import { ScenarioService } from '@services';
+
+@UntilDestroy()
+@Component({
+  selector: 'app-treatmentplan-about-tab',
+  standalone: true,
+  imports: [AsyncPipe, DecimalPipe, NgIf],
+  templateUrl: './treatmentplan-about-tab.component.html',
+  styleUrl: './treatmentplan-about-tab.component.scss',
+})
+export class TreatmentplanAboutTabComponent implements OnInit {
+  constructor(
+    private treatmentsState: TreatmentsState,
+    private scenarioService: ScenarioService
+  ) {}
+  summary$ = this.treatmentsState.summary$;
+  scenarioId = this.treatmentsState.getScenarioId();
+  areaAcres: number | undefined = undefined;
+  standSize: string | undefined = undefined;
+
+  ngOnInit(): void {
+    if (this.scenarioId) {
+      this.loadScenario(this.scenarioId);
+    }
+  }
+
+  loadScenario(scenarioId: number) {
+    this.scenarioService
+      .getScenario(scenarioId.toString())
+      .pipe(untilDestroyed(this))
+      .subscribe((scenario: Scenario) => {
+        this.standSize = scenario.configuration?.stand_size;
+        //TODO: confirm that this is actually the acres value we want to display- not planning area acres
+        // this is calculating from scenario_result, just as we do in the result table
+        this.areaAcres = scenario.scenario_result?.result.features
```

Comment:
not sure I understand this one, but are you confirming this as part of this pr, or as a follow up? If follow up, remember to add a story or ticket for it so we dont forget about this :D